notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            super().__init__(model.config)  # pass config from the original model
</pre><p>The same issue. You could save a config for the particular model and load it here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Receiving config for pretrained model was added.